### PR TITLE
@features/restaurants

### PR DESCRIPTION
**Pull request by rambhagavan**

### CODE REVIEW

File: admin-ui/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "json-viewer-js": "^1.0.8",
     "lodash": "^4.17.21",
     "node-sass": "^7.0.3",
+    "prettier": "^3.2.4",
```
Comment (subhamsahu) [inline admin-ui/package.json:29]: Add as dev dependency

File: admin-ui/src/app/components/Dialogs/ConfirmDialog.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
 const ConfirmDialog = ({ open, onClose, onConfirm, title, content }) => {
   return (
     <Dialog open={open} onClose={onClose}>
-      <DialogTitle>{title}</DialogTitle>
+       <DialogTitle>{title}</DialogTitle>
```
Comment (subhamsahu) [inline admin-ui/src/app/components/Dialogs/ConfirmDialog.js:15]: Do proper formatting

File: admin-ui/src/app/pages/Restaurant/components/CreateForm.js [hunk 3]
```diff
@@ -16,15 +16,25 @@ import { BACKEND_URL } from '../../../core/constants';
 import Chip from '@mui/material/Chip';
 import Stack from '@mui/material/Stack';
 import AddIcon from '@mui/icons-material/Add';
-
+// adding adhar_number
+// adding banking details(ifsccode and account number)
```
Comment (subhamsahu) [inline admin-ui/src/app/pages/Restaurant/components/CreateForm.js:20]: Avoid Unnecessary Comments

File: server/app/controllers/auth.js [hunk 4]
```diff
@@ -79,7 +79,7 @@ const login = async (req, res) => {
 
 const register = async (req, res) => {
     try {
-        const { email, firstName, lastName, password } = req.body;
+        const { email, firstName, lastName, password ,role} = req.body;
```
Comment (subhamsahu) [inline server/app/controllers/auth.js:82]: Remove Role

File: server/app/controllers/upload.js [hunk 5]
```diff
@@ -5,8 +5,13 @@ const fs = require("fs");
 const {cloudinaryUploadImg,cloudinaryDeleteImg} = require('../utils/cloudinary');
 
 const upload = async (req, res) => {
+ 
     try {
+        if (!req.files || req.files.length === 0) {
+            return res.status(400).json({ success: false, message: 'No files uploaded.' });
```
Comment (subhamsahu) [inline server/app/controllers/upload.js:11]: Do testing once all PR merged

File: server/app/routes/api/upload.js [hunk 6]
```diff
@@ -9,8 +9,7 @@ const auth = require('../../middleware/auth');
 // fetch all addresses api for a user
 router.post('/',
     auth(),
-    uploadPhoto.array("images", 10),
-    productImgResize,
+    uploadPhoto.array("images", 10),productImgResize,
```
Comment (subhamsahu) [inline server/app/routes/api/upload.js:12]: Revert to original

**Review by rambhagavan — COMMENTED** [review]
code formatting done and removed some unnecessary comments

**Pull request closed without merging**